modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldServiceImpl.java

```diff
@@ -123,6 +123,20 @@ externalReferenceCode, objectFieldId, getUserId(),
 			objectField.isSystem(), objectFieldSettings);
 	}
 
+	@Override
+	public ObjectField updateRequired(long objectFieldId, boolean required)
+		throws PortalException {
+
+		ObjectField objectField = objectFieldPersistence.findByPrimaryKey(
+			objectFieldId);
+
+		_objectDefinitionModelResourcePermission.check(
+			getPermissionChecker(), objectField.getObjectDefinitionId(),
+			ActionKeys.UPDATE);
+
+		return objectFieldLocalService.updateRequired(objectFieldId, required);
+	}
+
```

Comment:
@G-Sa please check if we really need this method in the `service`.

Reply:
I created it only as a failsafe for security reasons, but its not needed. Ill drop it